src/styles/less/InstructionsModal.less

```diff
@@ -11,14 +11,58 @@
     padding: 5%;
     min-width: 100%;
 
-    p {
-      font-size: 2.5rem;
-    }
     .accept-button {
       .orangeButtonStyle();
       position: absolute;
       right: 8%;
       top: 82.33%;
     }
+    .ant-modal-body {
+      p {
+        font-size: 3rem;
+      }
+    }
   }
 }
+
+@media @tablet {
+  .instructions-modal {
+    .ant-modal-content {
+        .accept-button {
+          width: 120px;
+        }
+        .ant-modal-body {
+          p {
+            font-size: 2.2rem;
+          }
+        }
+    }
+  }
+}
+
+@media @mobile {
+  .instructions-modal {
+    .ant-modal-content {
+        .accept-button {
+          width: 100px;
+        }
+        .ant-modal-body {
+          p {
+            font-size: 1.8rem;
+          }
+        }
+    }
+  }
+}
+
+@media @xs {
+  .instructions-modal {
+    .ant-modal-content {
+        .ant-modal-body {
+          p {
+            font-size: 1.4rem;
+          }
+        }
+    }
+  }
+}
```

Comment:
Did you see any reason that this modals CSS needed its own file? Perhaps we can put it into the main file with all the other CSS for the page it pops out from. We could just separate it out with some text stating what the following CSS relates to.

Reply:
At the moment, it appears to be a unique modal (as in, there are no other modals within the application that follow a similar look/layout.) Considering that, I think it's a good idea to separate it into its own LESS file as has been done. 
Furthermore, the modals that we have spoken about implementing for the Forms (update settings, child name, credit card, ETC) will follow a different layout, so I think we will have to be sure to keep that LESS styling separate from this InstructionModal if we would like to follow the current design pattern of the application.